docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -32,6 +32,8 @@ Fleet UI:
 
 2. Under **End user authentication**, enter your IdP credentials and select **Save**.
 
+Note: If you have already configured your SSO settings for [Fleet admin authentication](https://fleetdm.com/docs/configuration/fleet-server-configuration#okta-idp-configuration), you will need to edit the settings in your identity provider to reference an additional Single Sign on URL. To do this for an Okta configuration, in the app "Advanced Settings"> "Other Requestable SSO URLs", set the URL to "https://fleetserver.com/api/v1/fleet/mdm/sso/callback".
```

Comment:
I thought the underlying problem was that we are missing the steps for configuration on the IdP side?

I think we have this documented [here](https://fleetdm.com/docs/configuration/fleet-server-configuration#identity-provider-idp-configuration). Maybe we point to this section of the docs and call out that you need to set the "Assertion Consumer Service" to `https://fleetserver.com/api/v1/fleet/mdm/sso/callback` instead?

Reply:
@noahtalerman the documentation you've reference is for the SSO setup specifically for admin logins. It would be odd to go to that documentation, as it doesn't have an impact on the EU SSO setup. This is a stop gap until SSO settings can be referenced in a single location, as requested in https://github.com/fleetdm/fleet/issues/13524

Reply:
> It would be odd to go to that documentation, as it doesn't have an impact on the EU SSO setup. 

@dherder what about users that don't have SSO setup for Fleet users (admins, maintainers, observers)? I think we want to guide them on how to set config on the IdP side.

(I guess we can add that in a follow up pass)

Reply:
@noahtalerman I added some additional clarification to best practices of using 2 separate SSO apps. I think we would avoid documenting the Okta UI config, as it seems to change quite often.